README.md

```diff
@@ -72,7 +71,18 @@ Here's how to invoke this example module in your projects
 ```hcl
 module "example" {
   source = "https://github.com/cloudposse/terraform-aws-config.git?ref=master"
-  example = "Hello world!"
+
+  create_sns_topic = true
+  create_iam_role  = true
+
+  managed_rules = {
```

Comment:
Let's implement a `catalog` in a follow up PR.

Reply:
oh, maybe we can't due to the custom parameters. Never mind.

Reply:
not saying we need to do it right now (adding `catalog`), but our module `terraform-yaml-config` supports templating, so all those YAML files in the `catalogs` are actually templates, so custom params can be templatized and supplied by other stack configuration

Reply:
Nice! I'll look into this and get familiar with the `terraform-yaml-config` module as well.